Automattic/wp-calypso
# Sanitize control characters

Related to #
p3btAN-2Xl-p2

## Proposed Changes

* Sanitization was needed for signup_url param



## Why are these changes being made?


* Improve handling of query params

## Testing Instructions



* Build this PR
* Visit `/log-in/en-gb?signup_url=/%0d%0a/evil.com&is_signup_existing_account=true&email_address=test@test.com`
* Ensure that the control chars are sanitized from the links. See the relevant post

## Pre-merge Checklist
